stdlib/asyncio/tasks.pyi

```diff
@@ -79,7 +80,9 @@ if sys.version_info >= (3, 12):
     _FutureLike: TypeAlias = Future[_T] | Awaitable[_T]
 else:
     _FutureLike: TypeAlias = Future[_T] | Generator[Any, None, _T] | Awaitable[_T]
+
 _TaskYieldType: TypeAlias = Future[object] | None
+_ThreadsafeCouroutineType: TypeAlias = Coroutine[Any, Any, _T] | CoroutineType[Any, Any, _T]
```

Comment:
`CoroutineType` is a subtype of the abstract base class `Coroutine`, so this Union will be (either eagerly or lazily) simplified by type checkers to `Coroutine`. I'd be inclined to just use `Coroutine` directly rather than having this alias